src/app/ui-components/selectFeeRate/feeItem.tsx

```diff
@@ -171,17 +181,30 @@ function FeeItem({
           <StyledSubText typography="body_medium_s" color={secondaryColor}>
             {time ?? `~${priorityTimeMap[priority]} mins`}
           </StyledSubText>
-          <StyledP
-            typography="body_medium_s"
-            color={secondaryColor}
-          >{`${feeRate} ${feeRateUnits}`}</StyledP>
+          {feeRateUnits && (
+            <StyledP
+              typography="body_medium_s"
+              color={secondaryColor}
+            >{`${feeRate} ${feeRateUnits}`}</StyledP>
+          )}
+          {!feeRateUnits && fiat && (
+            <StyledP typography="body_medium_s" color={secondaryColor}>
+              <NumericFormat
+                value={fiat}
+                displayType="text"
+                prefix={`~${currencySymbolMap[fiatUnit]}`}
+                thousandSeparator
+                renderText={(value: string) => `${value} ${fiatUnit}`}
+              />
+            </StyledP>
```

Comment:
This code is duplicated lower down with an inverse display logic. Please refactor to not have duplicate code.